pymatviz/ptable.py

```diff
@@ -503,7 +355,7 @@ def add_child_plots(
                 continue
 
             # Call child plotter
-            child_plotter(ax, plot_data, **child_kwargs)
+            child_plotter(ax, plot_data, tick_kwargs=tick_kwargs, **child_kwargs)  # type: ignore[call-arg]
```

Comment:
Not sure why `mypy` reported:
```python
pymatviz/ptable.py:358: error: Unexpected keyword argument "tick_kwargs"  [call-arg]
```

Reply:
[have a look at the docs for type-hinting `Callable`](https://docs.python.org/3/library/typing.html#annotating-callable-objects). it's quite restricted on what signatures it can model. to be maximally permissive in the inputs while still constraining the output, you can use `Callable[..., SomeReturnType]`

Reply:
Beautiful. That's a wonderful fix!